### PR TITLE
Portfolio continuation

### PR DESCRIPTION
**Pull request by essiewanjiru99**
I used the following to continue building my portfolio;

- Added index.css
- Added index.html
- l used linters

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -38,5 +38,181 @@ <h1 class="lines"> Hey there <br>
           <img src="images/vector.svg" alt="normalbottun">
         </div>
     </section>
+    
+
+    <section class="line-break">
```
Comment (Kossi-stack) [inline index.html:43]: - [ ]Kindly make sure to have only one section for my recent works. For clean code purposes

File: index.html [hunk 2]
```diff
@@ -38,5 +38,181 @@ <h1 class="lines"> Hey there <br>
...
+      <div>
+      <h2>My Recent <br> Works</h2>
+      </div>
+    </section>
+
```
Comment (Kossi-stack) [inline index.html:48]: - [ ] Kindly remove the empty line in your code it help you to have clean code

File: index.html [hunk 3]
```diff
@@ -38,5 +38,181 @@ <h1 class="lines"> Hey there <br>
...
+
+    <section class="section-placeholder">
+      <div  class="placeholder-container">
+          <img src="images/placeholder.svg" alt="placeholder" class="placeholder">
+          <div class="text-container">
+            <h1>Multi-Post Stories <br> Gain+Glory</h1>
```
Comment (Kossi-stack) [inline index.html:54]: - [ ] Kindly use another heading like h2 ...... hx for subtitles. Use h1 only for the main Title.

File: index.html [hunk 4]
```diff
@@ -38,5 +38,181 @@ <h1 class="lines"> Hey there <br>
...
+              <li class="text-java">JavaScript</li>
+              <li class="text-html">HTML</li>
+            </ul>
+            <div class="button1"><button class="button-text" type="button">See project </button></div>
+          </div>
+      </div>
+      <div  class="placeholder-container">
+          <img src="images/placeholder.svg" alt="placeholder" class="placeholder">
+          <div class="text-container">
+            <h1>Multi-Post Stories <br> Gain+Glory</h1>
```
Comment (Kossi-stack) [inline index.html:67]: - [ ] Kindly use another heading like h2 ...... hx for subtitles. Use h1 only for the main Title.

File: index.html [hunk 5]
```diff
@@ -38,5 +38,181 @@ <h1 class="lines"> Hey there <br>
...
+            <li class="text-java">JavaScript</li>
+            <li class="text-html">HTML</li>
+          </ul>
+          <div class="button1"><button class="button-text" type="button">See project </button></div>
+        </div>
+    </div>
+    </section>
+      <section class="about-container">
+        <div class="text-aline">
+
```
Comment (Kossi-stack) [inline index.html:132]: - [ ] Kindly remove the empty line in your code it helps you to have clean code

File: index.html [hunk 6]
```diff
@@ -38,5 +38,181 @@ <h1 class="lines"> Hey there <br>
...
+        </div>
+  
+         <div class="section6">
+          <article class="qoute">
+            <div class="holder-text">
+              <img src="images/holder.svg" alt="holder">
+              <div class="holder-text1">
+                <img src="images/ellipse.svg" alt="holder">
+                <div class="Language">
+                  <p>Language</p>
```
Comment (Kossi-stack) [inline index.html:168]: - [ ] Kindly make sure the technology categories ("Languages", "Frameworks" and "Skills") are in a ul, each of them in a li

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @essiewanjiru99,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ Linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

 - [ ] Kindly edit the PR's title to something more descriptive. it can be "Portfolio: finish mobile version Project".
 - [ ] Kindly make sure next time you make your Readme file available in the branch for review purposes, remove the live demo section since it is not available.
 
 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 7]
```diff
@@ -38,5 +38,169 @@ <h1 class="lines"> Hey there <br>
...
+                      <div class="Language">
+                        <p>Skills</p>
+                      </div>
+                      <ul class="media">
+                        <li>Database Management</li>
+                        <li>Version Controle</li>
+                        <li>CLI</li>
+                        <li>WEb Development</li>
+                        <li>API Design</li>
+                      <ul>
```
Comment (NatiG25) [inline index.html:198]: - [ ] As mentioned by the previous reviewer, you should have:
- Your technology categories ("Languages", "Frameworks" and "Skills") in a `ul`, & each of them in a `li`. 
- The technologies in each category(e.g., "JavasScript", "HTML" and "CSS") should also be in a `ul` and each of them in a `li`

**Review by NatiG25 — CHANGES_REQUESTED** [review]
## :yellow_circle: CHANGES REQUIRED :yellow_circle: 

## Hi @essiewanjiru99 :wave:, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Highlights
:heavy_check_mark: Good job! Using only one `h1`.
:heavy_check_mark: Remove empty lines.



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Owusu-Desmond — APPROVED** [review]
Hi @essiewanjiru99 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**